pyspark/bigdl/keras/converter.py

```diff
@@ -113,6 +113,15 @@ def to_bigdl_weights(class_name, weights):
     @staticmethod
     def get_bigdl_weights_from_klayer(klayer):
         # we should use get_weights instead of klayer.weights
+        if isinstance(klayer, keras.layers.wrappers.TimeDistributed):
```

Comment:
put this logic into weigthconverter for Timedistributed?

Reply:
Sure.

Reply:
def convert_timedistributed(weights):
Since the converter function only takes one argument, ie weights from keras, how can we do this? If we want to put this logic into this function, then we probably need one more argument klayer??

Reply:
@zhichao-li ?